ckanext/unaids/i18n/fr/LC_MESSAGES/ckanext-unaids.po

```diff
@@ -686,3 +670,49 @@ msgstr "Mes inscriptions"
 #~ msgid "Voluntary Medical Male Circumcision (VMMC) Data"
 #~ msgstr "Données de circoncision masculine médicale volontaire"
 
+#~ msgid ""
+#~ "Added field <q>{key}</q> with value "
+#~ "<q>{value}</q> to resource {resource_link} in"
+#~ " {pkg_link}"
+#~ msgstr ""
+
+#~ msgid "Added field <q>{key}</q> to resource {resource_link} in {pkg_link}"
+#~ msgstr ""
+
+#~ msgid "Added the following fields to resource {resource_link} in {pkg_link}"
+#~ msgstr ""
+
+#~ msgid "{key} with value {value}"
+#~ msgstr ""
+
+#~ msgid "{key}"
+#~ msgstr ""
+
+#~ msgid "Removed field <q>{key}</q> from resource {resource_link} in {pkg_link}"
+#~ msgstr ""
+
+#~ msgid ""
+#~ "Removed the following fields from "
+#~ "resource {resource_link} in {pkg_link}"
+#~ msgstr ""
+
+#~ msgid ""
+#~ "Changed value of field <q>{key}</q> of"
+#~ " resource {resource_link} to <q>{new_val}</q> "
+#~ "(previously <q>{old_val}</q>) in {pkg_link}"
+#~ msgstr ""
+
+#~ msgid ""
+#~ "Changed value of field <q>{key}</q> to"
+#~ " <q>{new_val}</q> in resource {resource_link} "
+#~ "in {pkg_link}"
+#~ msgstr ""
+
+#~ msgid ""
+#~ "Removed the value of field <q>{key}</q>"
+#~ " in resource {resource_link} in {pkg_link}"
+#~ msgstr ""
+
+#~ msgid "No fields were updated. See the metadata diff for more details."
+#~ msgstr ""
```

Comment:
I think that those lines are translation strings that got removed from the code.
You should review them and delete the lines if those strings were actually removed fromt he code base.

Reply:
Well spotted!